strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/InMemoryUserService.java

```diff
@@ -41,7 +41,8 @@
     private SecurityTokenProvider tokenProvider;
 
     @Override
-    public Users getUsers()
+    public Users getUsers(Long skip,
```

Comment:
@anki2189 we need `skip` and `limit` support here as well. Could you please also make this parameters required?

Reply:
Fixed.